neuromation/cli/formatters/jobs.py

```diff
@@ -288,10 +288,10 @@ def from_job(
             when = job.history.finished_at
         assert when is not None
         return cls(
-            id=job.id,
+            id=style(job.id, bold=True),
```

Comment:
You can use `bold()` defined at the top of this file.

Reply:
thanks!